### PR TITLE
Fixed capacitor integration

### PR DESCRIPTION
**Pull request by Proper-Job**
Fixed capacitor integration by removing the android-biometric.aar file and replacing it with a maven import.

I tested these changes using cordova and capacitor on real Android devices.

### CODE REVIEW

**Comment by exxbrain** [comment]
@Proper-Job, @NiklasMerz, it looks like a better solution. I tried to add sources to the plugin but it lead me to several problems. It is hard to resolve them.

File: README.md [hunk 1]
```diff
@@ -70,6 +70,20 @@ Get the latest development version. *Not recommended!*
...
+**Android integration (Capacitor and Cordova)**
+
+In you top level (project) ``build.gradle`` file add the maven repository url below.
+This will pull in the required androidx.biometric library adapted for non-androidx apps.
+
+```java
+allprojects {
+    repositories {
+        ...
+        maven { url 'https://jitpack.io' }
```
Comment (exxbrain) [inline README.md]: Adding this to the top level is not necessary. You can use src/android/build.gradle to link with jitpack.io. So user will not need to make any additional configuration steps.

Comment (Proper-Job) [reply README.md]: Hi @exxbrain,
I was wondering if there was a way to do that. Thank you for your suggestion. I will make the requested changes once I'm back in the office on Monday.

File: src/android/build.gradle [hunk 2]
```diff
@@ -8,7 +8,7 @@ dependencies {
      *
```
Comment (exxbrain) [inline src/android/build.gradle]: This must help:
```
repositories {
         maven { url 'https://jitpack.io' }
}
```

Comment (exxbrain) [reply src/android/build.gradle]: @Proper-Job, does it help?

Comment (Proper-Job) [reply src/android/build.gradle]: It works with capacitor this way but not with cordova. Debugging right now.

Comment (Proper-Job) [reply src/android/build.gradle]:
```groovy
allprojects {
    repositories {
        maven { url 'https://jitpack.io' }
    }
}
```
seems to do the trick for both capacitor and cordova, though.

Comment (Proper-Job) [reply src/android/build.gradle]: @exxbrain I pushed your change request. I will test this again with a clean cordova and capacitor install.

Comment (Proper-Job) [reply src/android/build.gradle]: @exxbrain @NiklasMerz I successfully tested cordova and capacitor integration with these latest changes on a real Android device. No additional configuration was necessary after installing the plugin.

**Comment by OlegIvaniv** [comment]
Great job guys 👌! Really looking forward to this fix :)

File: src/android/build.gradle [hunk 3]
```diff
@@ -1,7 +1,5 @@
-allprojects {
```
Comment (Proper-Job) [inline src/android/build.gradle]: @exxbrain in my tests the allprojects directive was required.

Comment (exxbrain) [reply src/android/build.gradle]: @Proper-Job, thanks! Could you please describe an issue happened without `allprojects`? I removed `allprojects` but probably I was wrong.

Comment (Proper-Job) [reply src/android/build.gradle]: I'm not exactly sure anymore, but I believe capacitor couldn't resolve the maven repository without the allprojects directive.

**Comment by exxbrain** [comment]
Some testing needed.

**Comment by Proper-Job** [comment]
@exxbrain are you sure that removing the ```allprojects``` directive from the gradle file works for cordova and capacitor?  Because I specifically added that directive because it wasn't working in my setup without it.

**Comment by exxbrain** [comment]
> @exxbrain are you sure that removing the `allprojects` directive from the gradle file works for cordova and capacitor? Because I specifically added that directive because it wasn't working in my setup without it.

@Proper-Job, I reverted it. Thank you!

**Comment by Proper-Job** [comment]
@exxbrain did you test the cordova and capacitor integrations?
Since the capacitor integration for the published version is broken right now, I think it would be good to release this ASAP.

**Comment by SSuarezBrooktec** [comment]
Testing yesterday and works properly!

**Comment by Proper-Job** [comment]
@exxbrain @NiklasMerz could we get a release soon?
Thank you

**Comment by exxbrain** [comment]
@Proper-Job @NiklasMerz, I've already approved this pull request.

**Comment by NiklasMerz** [comment]
Thank you all. I will get to do the next release, soon.

**Comment by NiklasMerz** [comment]
Thank you all. Version 3.0.1 is now published

**Comment by jarekskuder** [comment]
Warning: com.exxbrain.android.biometric.BiometricManager: can't find referenced class android.hardware.biometrics.BiometricManager
Warning: com.exxbrain.android.biometric.BiometricManager$Api29Impl: can't find referenced class android.hardware.biometrics.BiometricManager

Still have this errors when building release apk on version 3.0.1

**Pull request merged**